docs/algorithms/kem/bike.md

```diff
@@ -11,6 +11,9 @@
 - **Ancestors of primary source**:
   - https://bikesuite.org/files/v5.0/Reference_Implementation.2022.10.04.1.zip
 
+## Advisories
+This implementation of BIKE is not constant time.
```

Comment:
Changing this only in the .MD and not the .YML will not survive the next execution of `copy_from_upstream.py`.